src/BoundsMap.js

```diff
@@ -0,0 +1,45 @@
+import isUndefined from 'lodash/lang/isUndefined';
+import { PropTypes } from 'react';
+import Leaflet from 'leaflet';
+
+import boundsType from './types/bounds';
+
+import CenterZoomMap from './Map';
+
+
+export default class BoundsMap extends CenterZoomMap {
+  static propTypes = {
+    bounds: boundsType,
+    children: PropTypes.oneOfType([
+      PropTypes.arrayOf(PropTypes.node),
+      PropTypes.node,
+    ]),
+    className: PropTypes.string,
+    id: PropTypes.string,
+    maxBounds: boundsType,
+    maxZoom: PropTypes.number,
+    minZoom: PropTypes.number,
+    style: PropTypes.object,
+  };
+
+  componentDidMount() {
+    super.componentDidMount();
+    if (!isUndefined(this.props.bounds)) {
+      this.leafletElement.fitBounds(this.props.bounds);
+    }
+  }
+
+  componentDidUpdate({ bounds: oldBounds }) {
```

Comment:
I think this would be better as `componentDidUpdate(props)`, as it makes things more clear to anybody that wants to extend `BoundsMap`.